furiosa/models/vision/efficientnet_v2_s/__init__.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+from typing import Any, Dict, List, Sequence, Tuple
```

Comment:
```suggestion
from typing import Any, Dict, List, Sequence, Tuple, Type
```

밑의 리뷰 커멘트에서 추가로 필요한 의존성입니다.